crates/pink-drivers/system/lib.rs

```diff
@@ -50,6 +50,11 @@ mod system {
             Err(Error::BadOrigin)
         }
 
+        #[ink(message)]
+        pub fn is_admin(&self, contract_id: AccountId) -> Result<bool> {
```

Comment:
Should we move it into the `trait System`?

Reply:
sure, let me do that

Reply:
Let me do it if you haven't time. OK?

Reply:
Yes, pls. Thx.